.changeset/nice-snakes-camp.md

```diff
@@ -0,0 +1,90 @@
+---
+"@salt-ds/core": minor
+---
+
+**_Theming and CSS updates_** with visual changes to Salt components:
+
+1. Minor changes in `Button` component width. `CTA` and `Primary` button text font weight changed from bold to semibold. Expect components containing Salt `Button` to have similar change.
+
+```diff
```

Comment:
I'm not sure we would show this diff as it might encourage changing the local `--button-fontWeight` token. I think explaining the difference of font weight as above would be enough

Reply:
good point - included the diff as wanted to redirect people to the new token to use since we deprecate 3 of them. Is this better?
```
- --salt-actionable-cta-fontWeight
- --salt-actionable-primary-fontWeight
- --salt-actionable-secondary-fontWeight
+ --salt-text-action-fontWeight
```